website/vault.erb

```diff
@@ -25,6 +25,10 @@
                             <a href="/docs/providers/vault/d/auth_backend.html">vault_auth_backend</a>
                         </li>
 
+                        <li<%= sidebar_current("docs-vault-datasource-ad-access-credentials") %>>
+                            <a href="/docs/providers/vault/d/ad_access_credentials.html">vault_ad_access_credentials</a>
+                        </li>
+
```

Comment:
Unfortunately this file is no longer used at all, and should be removed in a separate PR